src/stratigraphy/util/coordinate_extraction.py

```diff
@@ -128,23 +113,23 @@ def __init__(self, document: fitz.Document):
         self.coordinate_keys = ["Koordinaten", "Koordinate", "coordinates", "coordinate", "coordonnés", "coordonnes"]
         # TODO: extend coordinate keys with other languages
 
-    def find_coordinate_key(self, text: str, allowed_operations: int = 3) -> str:
+    def find_coordinate_key(self, text: str, allowed_errors: int = 3) -> str:
         """Finds the location of a coordinate key in a string of text.
 
         This is is useful to reduce the text within which the coordinates are searched. If the text is too large
         false positive (found coordinates that are no coordinates) are more likely.
 
         Args:
             text (str): Arbitrary string of text.
-            allowed_operations (int, optional): The maximum number of allowed operations to consider a key contained
-                                                in text. Defaults to 3.
+            allowed_errors (int, optional): The maximum number of errors to consider a key contained
+                                            in text. An error is a character that is different. Defaults to 3.
```

Comment:
Is this sufficient for you? There is not soo much documentation on that.

The best is to search for fuzzy matching with regex. There are some stackoverflow threads talking about it, but the official documentation is not very helpful.

Reply:
But these "errors" are the same as for Levenshtein distance, is that correct? In that case, it would be clearer to write something like: "The maximum Levenshtein distance from a true coordinate key to a phrase in the text, for that phrase to be accepted as a valid coordinate key. Defaults to 3 (guestimation; no optimisation done yet)."